OneCommunityGlobal/HighestGoodNetworkApp
# shrey_conditional_render_google_grayed_icon

# Description
Conditionally render google doc icon to only show if profile has google doc link 

- Currently, on the weekly summary reports page to the right of the name value this icon is present: This icon is there whether the profile has a google doc link or not, which gives the appearance of it being there.
-  To make this page more user friendly, I suggest implementing a conditional operator to render the google doc icon with a different appearance if the profile does not have a google doc link. Maybe a grayed out version?


## Main changes explained:
- Grayed out the google doc icon, if there is not google doc link present.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to dashboard→ Reports→ Weekly Summaries Reports→…
5. verify that google doc icon is grayed out for users who have no google link

## Screenshots or videos of changes:
Before:
All the google doc icons were colored for all users.
<img width="1429" alt="Google_doc_colored" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118795427/5b524fff-0ea3-44a3-ab1d-66347e4deff0">

<img width="1440" alt="Screenshot 2023-08-19 at 4 23 57 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118795427/cbf7bcf8-cfbd-4908-ae4d-584ddb5960e6">

After:
Only the user with google doc link present have the icon in colored
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118795427/ef79d3ed-09af-4d04-bdf0-b2298d43a3c4
